pytket/extensions/pyquil/backends/forest.py

```diff
@@ -108,7 +108,14 @@ class ForestBackend(Backend):
     _supports_counts = True
     _supports_contextual_optimisation = True
     _persistent_handles = True
-    _GATE_SET = {OpType.CZ, OpType.Rx, OpType.Rz, OpType.Measure, OpType.Barrier}
+    _GATE_SET = {
+        OpType.CZ,  #
+        OpType.Rx,  #
+        OpType.Rz,  #
+        OpType.Measure,  #
+        OpType.Barrier,  #
```

Comment:
Why the comment markers?

Reply:
I was using the wrong formatter version, solved now. See [641de02](https://github.com/CQCL/pytket-pyquil/pull/37/commits/641de0237edef79ce371fb6065364990f5a73cb7)